tests/main/views/test_buyers.py

```diff
@@ -106,7 +106,7 @@ def setup_method(self, method):
     def test_draft_briefs_section(self, data_api_client, find_briefs_mock):
         data_api_client.find_briefs.return_value = find_briefs_mock
 
-        res = self.client.get("/buyers")
+        res = self.client.get("/buyers/requirements/digital-outcomes-and-specialists")
```

Comment:
I wonder if we could use url_for here, it might not fit in with existing tests but it'd save us some grief down the line